components/Base/Button.vue

```diff
@@ -6,55 +6,42 @@ import ArrowRight from '~/assets/icons/arrow-right.svg?skipsvgo';
 
 const props = withDefaults(
   defineProps<{
-    cta?: boolean;
-    variant?: 'fill' | 'outline' | 'featured-link';
-    type?: 'regular' | 'round' | 'square';
+    variant?:
+      | 'fill'
+      | 'outline'
+      | 'text'
+      | 'featured-link'
```

Comment:
what is the difference between `featured-link` and `text`? 🤔

Reply:
Not much to be honest for example on home page we have links with arrow that has different font weight. This can be removed?

Reply:
on second thought. Not necessary and removed